docs/learn/advanced/overview.md

```diff
@@ -77,6 +77,12 @@ In addition to executing stream transactions according to StreamType logic, Cera
 - **Query responses:** Ceramic nodes respond to stream queries from clients. If the node has the stream pinned, it will return the response; if not, it will ask the rest of the network for the stream over libp2p and then return the response.
 - **Broadcasting transactions:** When a Ceramic node successfully performs a transaction on a stream, it broadcasts this transaction out the rest of the network over libp2p so other nodes also pinning this stream can update their StreamState to reflect this new transaction.
 
+### Components of a Ceramic Node
+
+A ceramic node requires several components in addition to the actual ceramic instance: an IPFS running instance and storages, a ceramic state store, and log storage.
```

Comment:
This is a bit confusing. Who is the intended audience and what level of abstraction are we aiming to give?

Is this information for node operators? Or is it to help people understand how the Ceramic system works?

If the latter than it should just be 3 components of the system:
- ceramic daemon
- ipfs
- ceramic anchor service

Reply:
Also diagram should reflect the abstraction accurately. This seems like a diagram for node operators and I think would be more confusing than helpful for general public--i.e. people will think Ceramic *relies on* S3